runironic.sh

```diff
@@ -2,6 +2,8 @@
 
 . /bin/configure-ironic.sh
 
+crudini --set /etc/ironic/ironic.conf DEFAULT host localhost
```

Comment:
Why doesn't this do the same thing that the change in runironic-conductor.sh does?

Reply:
When ironic-conductor is run in a separate container we have to check to see whether it is configured to run in the same pod as ironic-api to see whether we should use localhost as the name of the conductor instance in the DB (if it's in a different pod, conductor gets the hostname from the OS).

When ironic-conductor and ironic-api are in the same container they can always communicate using the localhost interface, so there seemed no point in copying and pasting all of that logic.

An alternative would be to move the setup for this option to configure-ironic.sh. I didn't do that because ironic-api also reads the `host` option for the profiler setup. I think it probably doesn't matter, but I didn't want to get into it.

Reply:
OK, I missed that this script was running both services in the same container.